

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>About &mdash; Cancer Predisposition Sequencing Reporter 0.4.0 documentation</title>
  

  
  
  
  

  

  
  
    

  

  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <link rel="author" title="About these documents" href="#" />
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Getting started" href="getting_started.html" />
    <link rel="prev" title="Welcome to Cancer Predisposition Sequencing Reporter’s documentation!" href="index.html" /> 

  
  <script src="_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="index.html" class="icon icon-home"> Cancer Predisposition Sequencing Reporter
          

          
          </a>

          
            
            
              <div class="version">
                0.4.0
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">About</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#what-is-the-cancer-predisposition-sequencing-preporter-cpsr">What is the Cancer Predisposition Sequencing Preporter (CPSR)?</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#cancer-predisposition-genes">Cancer predisposition genes</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#example-report">Example report</a></li>
<li class="toctree-l2"><a class="reference internal" href="#docker-based-technology">Docker-based technology</a></li>
<li class="toctree-l2"><a class="reference internal" href="#contact">Contact</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="getting_started.html">Getting started</a></li>
<li class="toctree-l1"><a class="reference internal" href="annotation_resources.html">CPSR annotation resources</a></li>
<li class="toctree-l1"><a class="reference internal" href="input.html">Input</a></li>
<li class="toctree-l1"><a class="reference internal" href="output.html">Output</a></li>
<li class="toctree-l1"><a class="reference internal" href="variant_classification.html">Variant classification (ACMG)</a></li>
<li class="toctree-l1"><a class="reference internal" href="CHANGELOG.html">CHANGELOG</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Cancer Predisposition Sequencing Reporter</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html">Docs</a> &raquo;</li>
        
      <li>About</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/about.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="about">
<h1>About<a class="headerlink" href="#about" title="Permalink to this headline">¶</a></h1>
<div class="section" id="what-is-the-cancer-predisposition-sequencing-preporter-cpsr">
<h2>What is the Cancer Predisposition Sequencing Preporter (CPSR)?<a class="headerlink" href="#what-is-the-cancer-predisposition-sequencing-preporter-cpsr" title="Permalink to this headline">¶</a></h2>
<p>The <em>Cancer Predisposition Sequencing Reporter (CPSR)</em> is a
computational workflow that <strong>interprets germline variants</strong> identified
from next-generation sequencing <strong>in the context of cancer
predisposition</strong>. The workflow is integrated with the framework that
underlies the <a class="reference external" href="https://github.com/sigven">Personal Cancer Genome Reporter
(PCGR)</a>, utilizing the Docker environment
for encapsulation of code and software dependencies. While <em>PCGR</em> is
intended for reporting and analysis of somatic variants detected in a
tumor, <em>CPSR</em> is intended for reporting and ranking of germline variants
in protein-coding genes that are implicated in cancer predisposition and
inherited cancer syndromes.</p>
<p><em>CPSR</em> accepts a query file with raw germline variant calls encoded in
the <a class="reference external" href="https://samtools.github.io/hts-specs/VCFv4.2.pdf">VCF</a> format
(i.e.&nbsp;analyzing SNVs/InDels). Furthermore, through the use several
different <em>virtual cancer predisposition gene panels</em> harvested from the
<a class="reference external" href="https://panelapp.genomicsengland.co.uk/">Genomics England PanelApp</a>,
the user can flexibly put a restriction on which genes and findings are
displayed in the cancer predisposition report.</p>
<p>The software performs extensive variant annotation on the selected
geneset and produces an interactive HTML report, in which the user can
investigate four types of variants:</p>
<ol class="arabic simple">
<li><strong>ClinVar variants</strong> - pre-classified variants according to a
five-level tier scheme (Pathogenic to Benign)</li>
<li><strong>Non-ClinVar variants</strong> - classified by CPSR according to a
five-level tier scheme (Pathogenic to Benign)</li>
<li><strong>Secondary findings (optional)</strong> - pathogenic ClinVar variants in
the ACMG recommended list for reporting of incidental findings</li>
<li><strong>GWAS hits (optional)</strong> - variants overlapping with previously
identified hits in genome-wide association studies (GWAS) of cancer
phenotypes (i.e.&nbsp;low to moderate risk conferring alleles), using
<a class="reference external" href="https://www.ebi.ac.uk/gwas/">NHGRI-EBI Catalog of published genome-wide association
studies</a> as the underlying source.</li>
</ol>
<p>The variant sets can be interactively explored and filtered further
through different types of filters (phenotypes, genes, variant
consequences, population MAF etc.). Importantly, the unclassified
non-ClinVar variants are assigned a <em>pathogenicity level</em> based on the
aggregation of scores according to previously established <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/25741868">ACMG
criteria</a>. The ACMG
criteria includes cancer-specific criteria, as outlined and specified in
several previous studies (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/29625052">Huang et al., Cell,
2018</a>; <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/28492532">Nykamp et al.,
Genet Med., 2017</a>;
<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/27153395">Maxwell et al., Am J Hum Genet.,
2016</a>; <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/27181684">Amendola et
al., Am J Hum Genet.,
2016</a>). See also
<a class="reference external" href="https://github.com/sigven/cpsr#related-work">Related work</a> below).</p>
<div class="section" id="cancer-predisposition-genes">
<h3>Cancer predisposition genes<a class="headerlink" href="#cancer-predisposition-genes" title="Permalink to this headline">¶</a></h3>
<p>The cancer predisposition report can show variants found in a number of
well-known cancer predisposition genes, and the specific set of genes
can be customized by the user by choosing any of the following <strong>virtual
gene panels (0 - 33)</strong>:</p>
<ul>
<li><p class="first"><strong>Panel 0 (default)</strong> is a comprehensive gene panel assembled through
known sources on cancer predisposition:</p>
<ul class="simple">
<li>A list of 152 genes that were curated and established within
TCGA’s pan-cancer study (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/29625052">Huang et al., Cell,
2018</a>)</li>
<li>A list of 107 protein-coding genes that has been manually curated
in COSMIC’s <a class="reference external" href="https://cancer.sanger.ac.uk/census">Cancer Gene Census
v88</a>,</li>
<li>A list of 148 protein-coding genes established by experts within
the Norwegian Cancer Genomics Consortium
(<a class="reference external" href="http://cancergenomics.no">http://cancergenomics.no</a>)</li>
</ul>
<p>The combination of the three sources resulted in a non-redundant set
of <a class="reference external" href="https://github.com/sigven/cpsr/blob/master/predisposition.md">209 protein-coding
genes</a>
of relevance for predisposition to tumor development.</p>
</li>
<li><p class="first"><strong>Panels 1 - 33</strong> are collected from cancer predisposition panels
assembled within the <a class="reference external" href="https://panelapp.genomicsengland.co.uk/">Genomics England
PanelApp</a>:</p>
<ul class="simple">
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/245/">1 = Adult solid tumours cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/391/">2 = Adult solid tumours for rare
disease</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/208/">3 = Bladder cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/166/">4 = Brain cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/55/">5 = Breast cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/259/">6 = Childhood solid tumours cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/244/">7 = Colorectal cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/271/">8 = Endometrial cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/167/">9 = Familial Tumours Syndromes of the central &amp; peripheral
Nervous
system</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/158/">10 = Familial breast
cancer</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/522/">11 = Familial
melanoma</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/318/">12 = Familial prostate
cancer</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/290/">13 = Familial
rhabdomyosarcoma</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/59/">14 = Haematological malignancies cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/115/">15 = Head and neck cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/254/">16 = Inherited colorectal cancer (with or without
polyposis)</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/171/">17 = Inherited non-medullary thyroid
cancer</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/143/">18 = Inherited ovarian cancer (without breast
cancer)</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/524/">19 = Inherited pancreatic
cancer</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/521/">20 = Inherited renal
cancer</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/97/">21 = Inherited phaeochromocytoma and
paraganglioma</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/133/">22 = Melanoma pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/36/">23 = Multiple endocrine
tumours</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/183/">24 = Neuroendocrine cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/117/">25 = Ovarian cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/86/">26 = Parathyroid
Cancer</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/17/">27 = Prostate cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/154/">28 = Renal cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/600/">29 = Rhabdoid tumour
predisposition</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/217/">30 = Sarcoma cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/421/">31 = Thyroid cancer pertinent cancer
susceptibility</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/243/">32 = Tumour predisposition - childhood
onset</a></li>
<li><a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/273/">33 = Upper gastrointestinal cancer pertinent cancer
susceptibility</a></li>
</ul>
</li>
</ul>
</div>
</div>
<div class="section" id="example-report">
<h2>Example report<a class="headerlink" href="#example-report" title="Permalink to this headline">¶</a></h2>
<ul class="simple">
<li><a class="reference external" href="http://folk.uio.no/sigven/example.cpsr.grch37.html">Cancer predisposition sequencing
report</a></li>
</ul>
</div>
<div class="section" id="docker-based-technology">
<h2>Docker-based technology<a class="headerlink" href="#docker-based-technology" title="Permalink to this headline">¶</a></h2>
<p>The CPSR workflow is developed using the <a class="reference external" href="https://www.docker.com/what-docker">Docker
technology</a>. The software is thus
packaged into isolated containers, in which the installation of all
software libraries/tools and required dependencies have been taken care
of. In addition to the bundled software, in the form of a Docker image,
the workflow needs to be attached with an <a class="reference external" href="annotation_resources.html">annotation data
bundle</a>.</p>
<p><img alt="image0" src="_images/docker-logo50.png" /></p>
</div>
<div class="section" id="contact">
<h2>Contact<a class="headerlink" href="#contact" title="Permalink to this headline">¶</a></h2>
<p><a class="reference external" href="mailto:sigven&#37;&#52;&#48;ifi&#46;uio&#46;no">sigven<span>&#64;</span>ifi<span>&#46;</span>uio<span>&#46;</span>no</a></p>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="getting_started.html" class="btn btn-neutral float-right" title="Getting started" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="index.html" class="btn btn-neutral" title="Welcome to Cancer Predisposition Sequencing Reporter’s documentation!" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2018-2019, Sigve Nakken.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'./',
            VERSION:'0.4.0',
            LANGUAGE:'None',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
      <script type="text/javascript" src="_static/jquery.js"></script>
      <script type="text/javascript" src="_static/underscore.js"></script>
      <script type="text/javascript" src="_static/doctools.js"></script>

  

  <script type="text/javascript" src="_static/js/theme.js"></script>

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>  

  <style>
    /* Sidebar header (and topbar for mobile) */
    .wy-side-nav-search, .wy-nav-top {
      background: #0B6121;
    }
    /* Sidebar */
    /*.wy-nav-side {*/
    /*  background: #ff0000;*/
    /*}*/
  </style>


</body>
</html>