

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Getting started &mdash; Cancer Predisposition Sequencing Reporter 0.6.2 documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
        <script src="_static/language_data.js"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <link rel="author" title="About these documents" href="about.html" />
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="CPSR annotation resources" href="annotation_resources.html" />
    <link rel="prev" title="About" href="about.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home"> Cancer Predisposition Sequencing Reporter
          

          
          </a>

          
            
            
              <div class="version">
                0.6.2
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="about.html">About</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Getting started</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#step-0-python">STEP 0: Python</a></li>
<li class="toctree-l2"><a class="reference internal" href="#step-1-install-pcgr-version-0-9-2">STEP 1: Install PCGR (version 0.9.2)</a></li>
<li class="toctree-l2"><a class="reference internal" href="#step-2-download-the-latest-release">STEP 2: Download the latest release</a></li>
<li class="toctree-l2"><a class="reference internal" href="#step-4-run-example">STEP 4: Run example</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="annotation_resources.html">CPSR annotation resources</a></li>
<li class="toctree-l1"><a class="reference internal" href="superpanel.html">CPSR superpanel (v3.0)</a></li>
<li class="toctree-l1"><a class="reference internal" href="input.html">Input</a></li>
<li class="toctree-l1"><a class="reference internal" href="output.html">Output</a></li>
<li class="toctree-l1"><a class="reference internal" href="variant_classification.html">Variant classification (ACMG/AMP)</a></li>
<li class="toctree-l1"><a class="reference internal" href="CHANGELOG.html">CHANGELOG</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Cancer Predisposition Sequencing Reporter</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html">Docs</a> &raquo;</li>
        
      <li>Getting started</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/getting_started.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="getting-started">
<h1>Getting started<a class="headerlink" href="#getting-started" title="Permalink to this headline">¶</a></h1>
<div class="section" id="step-0-python">
<h2>STEP 0: Python<a class="headerlink" href="#step-0-python" title="Permalink to this headline">¶</a></h2>
<p>An installation of Python (version <em>3.6</em>) is required to run CPSR. Check
that Python is installed by typing <code class="docutils literal notranslate"><span class="pre">python</span> <span class="pre">--version</span></code> in your terminal
window.</p>
<p><strong>IMPORTANT NOTE</strong>: STEP 1 &amp; 2 below outline installation guidelines for
running CPSR with Docker. If you want to install and run CPSR without
the use of Docker (i.e. through Conda), follow <a class="reference external" href="https://github.com/sigven/cpsr/tree/master/conda_pkg/README.md">these
instructions</a></p>
</div>
<div class="section" id="step-1-install-pcgr-version-0-9-2">
<h2>STEP 1: Install PCGR (version 0.9.2)<a class="headerlink" href="#step-1-install-pcgr-version-0-9-2" title="Permalink to this headline">¶</a></h2>
<p>Make sure you have a working installation of PCGR (<strong>version 0.9.2</strong>)
and the accompanying data bundle(s) (walk through <a class="reference external" href="https://github.com/sigven/pcgr#getting-started">steps
1-2</a>).</p>
</div>
<div class="section" id="step-2-download-the-latest-release">
<h2>STEP 2: Download the latest release<a class="headerlink" href="#step-2-download-the-latest-release" title="Permalink to this headline">¶</a></h2>
<p>Download the <a class="reference external" href="https://github.com/sigven/cpsr/releases/tag/v0.6.2">0.6.2
release</a> of
<em>cpsr</em> (run script)</p>
</div>
<div class="section" id="step-4-run-example">
<h2>STEP 4: Run example<a class="headerlink" href="#step-4-run-example" title="Permalink to this headline">¶</a></h2>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>usage: cpsr.py -h [options]  --input_vcf &lt;INPUT_VCF&gt; --pcgr_dir &lt;PCGR_DIR&gt; --output_dir &lt;OUTPUT_DIR&gt; --genome_assembly  &lt;GENOME_ASSEMBLY&gt; --sample_id &lt;SAMPLE_ID&gt;

Cancer Predisposition Sequencing Reporter - report of clinically significant cancer-predisposing germline variants

Required arguments:
--input_vcf INPUT_VCF
                VCF input file with germline query variants (SNVs/InDels).
--pcgr_dir PCGR_DIR   Directory that contains the PCGR data bundle directory, e.g. ~/pcgr-0.9.2
--output_dir OUTPUT_DIR
                Output directory
--genome_assembly {grch37,grch38}
                Genome assembly build: grch37 or grch38
--sample_id SAMPLE_ID
                Sample identifier - prefix for output files

Panel options:
--panel_id VIRTUAL_PANEL_ID
                Comma-separated string with identifier(s) of predefined virtual cancer predisposition gene panels,
                choose any combination of the following identifiers:
                0 = CPSR exploratory cancer predisposition panel
                (n = 335, Genomics England PanelApp / TCGA Germline Study / Cancer Gene Census / Other)
                1 = Adult solid tumours cancer susceptibility (Genomics England PanelApp)
                2 = Adult solid tumours for rare disease (Genomics England PanelApp)
                3 = Bladder cancer pertinent cancer susceptibility (Genomics England PanelApp)
                4 = Brain cancer pertinent cancer susceptibility (Genomics England PanelApp)
                5 = Breast cancer pertinent cancer susceptibility (Genomics England PanelApp)
                6 = Childhood solid tumours cancer susceptibility (Genomics England PanelApp)
                7 = Colorectal cancer pertinent cancer susceptibility (Genomics England PanelApp)
                8 = Endometrial cancer pertinent cancer susceptibility (Genomics England PanelApp)
                9 = Familial Tumours Syndromes of the central &amp; peripheral Nervous system (Genomics England PanelApp)
                10 = Familial breast cancer (Genomics England PanelApp)
                11 = Familial melanoma (Genomics England PanelApp)
                12 = Familial prostate cancer (Genomics England PanelApp)
                13 = Familial rhabdomyosarcoma (Genomics England PanelApp)
                14 = GI tract tumours (Genomics England PanelApp)
                15 = Genodermatoses with malignancies (Genomics England PanelApp)
                16 = Haematological malignancies cancer susceptibility (Genomics England PanelApp)
                17 = Haematological malignancies for rare disease (Genomics England PanelApp)
                18 = Head and neck cancer pertinent cancer susceptibility (Genomics England PanelApp)
                19 = Inherited MMR deficiency (Lynch syndrome) - Genomics England PanelApp
                20 = Inherited non-medullary thyroid cancer (Genomics England PanelApp)
                21 = Inherited ovarian cancer (without breast cancer) (Genomics England PanelApp)
                22 = Inherited pancreatic cancer (Genomics England PanelApp)
                23 = Inherited polyposis (Genomics England PanelApp)
                24 = Inherited predisposition to acute myeloid leukaemia (AML) - Genomics England PanelApp
                25 = Inherited predisposition to GIST (Genomics England PanelApp)
                26 = Inherited renal cancer (Genomics England PanelApp)
                27 = Inherited phaeochromocytoma and paraganglioma (Genomics England PanelApp)
                28 = Melanoma pertinent cancer susceptibility (Genomics England PanelApp)
                29 = Multiple endocrine tumours (Genomics England PanelApp)
                30 = Multiple monogenic benign skin tumours (Genomics England PanelApp)
                31 = Neuroendocrine cancer pertinent cancer susceptibility (Genomics England PanelApp)
                32 = Neurofibromatosis Type 1 (Genomics England PanelApp)
                33 = Ovarian cancer pertinent cancer susceptibility (Genomics England PanelApp)
                34 = Parathyroid Cancer (Genomics England PanelApp)
                35 = Prostate cancer pertinent cancer susceptibility (Genomics England PanelApp)
                36 = Renal cancer pertinent cancer susceptibility (Genomics England PanelApp)
                37 = Rhabdoid tumour predisposition (Genomics England PanelApp)
                38 = Sarcoma cancer susceptibility (Genomics England PanelApp)
                39 = Sarcoma susceptibility (Genomics England PanelApp)
                40 = Thyroid cancer pertinent cancer susceptibility (Genomics England PanelApp)
                41 = Tumour predisposition - childhood onset (Genomics England PanelApp)
                42 = Upper gastrointestinal cancer pertinent cancer susceptibility (Genomics England PanelApp)

--custom_list CUSTOM_LIST
                Provide custom list of genes from virtual panel 0 (single-column txt file with Ensembl gene identifiers),
                alternative to predefined panels provided with --panel_id)
--custom_list_name CUSTOM_LIST_NAME
                Set name for custom made panel/list (single word - no whitespace), will be displayed in the report
--diagnostic_grade_only
                For panel_id&#39;s 1-42 (Genomics England PanelApp) - consider genes with a GREEN status only, default: False

VEP options:
--vep_n_forks VEP_N_FORKS
                Number of forks (option &#39;--fork&#39; in VEP), default: 4
--vep_buffer_size VEP_BUFFER_SIZE
                Variant buffer size (variants read into memory simultaneously, option &#39;--buffer_size&#39; in VEP)
                - set lower to reduce memory usage, default: 500
--vep_pick_order VEP_PICK_ORDER
                Comma-separated string of ordered transcript properties for primary variant pick
                ( option &#39;--pick_order&#39; in VEP), default: canonical,appris,biotype,ccds,rank,tsl,length,mane
--vep_no_intergenic   Skip intergenic variants during processing (option &#39;--no_intergenic&#39; in VEP), default: False

vcfanno options:
--vcfanno_n_proc VCFANNO_N_PROC
                Number of vcfanno processes (option &#39;-p&#39; in vcfanno), default: 4

Other options:
--force_overwrite     By default, the script will fail with an error if any output file already exists.
                You can force the overwrite of existing result files by using this flag, default: False
--version             show program&#39;s version number and exit
--basic               Run functional variant annotation on VCF through VEP/vcfanno, omit Tier assignment/report generation (STEP 4), default: False
--no_vcf_validate     Skip validation of input VCF with Ensembl&#39;s vcf-validator, default: False
--docker_uid DOCKER_USER_ID
                Docker user ID. Default is the host system user ID. If you are experiencing permission errors,
                try setting this up to root (`--docker_uid root`), default: None
--no_docker           Run the CPSR workflow in a non-Docker mode, default: False
--preserved_info_tags PRESERVED_INFO_TAGS
                Comma-separated string of VCF INFO tags from query VCF that should be kept in CPSR output TSV
--report_theme {default,cerulean,journal,flatly,readable,spacelab,united,cosmo,lumen,paper,sandstone,simplex,yeti}
                Visual report theme (rmarkdown), default: default
--report_nonfloating_toc
                Place the table of contents (TOC) at the top of the HTML report, not floating at the left, default: False
--report_table_display {full,light}
                Set the level of detail/comprehensiveness in interactive datables of HTML report, very comprehensive (option &#39;full&#39;) or slim/focused (&#39;light&#39;)
--ignore_noncoding    Do not list non-coding variants in HTML report, default: False
--secondary_findings  Include variants found in ACMG-recommended list for secondary findings (v3.0), default: False
--gwas_findings       Report overlap with low to moderate cancer risk variants (tag SNPs) identified from genome-wide association studies, default: False
--gwas_p_value GWAS_P_VALUE
                Required p-value for variants listed as hits from genome-wide association studies, default: 5e-06
--pop_gnomad {afr,amr,eas,sas,asj,nfe,fin,global}
                Population source in gnomAD used for variant frequency assessment (ACMG classification), default: nfe
--maf_upper_threshold MAF_UPPER_THRESHOLD
                Upper MAF limit (gnomAD global population frequency) for variants to be included in the report, default: 0.9
--classify_all        Provide CPSR variant classifications (TIER 1-5) also for variants with exising ClinVar classifications in output TSV, default: False
--clinvar_ignore_noncancer
                Ignore (exclude from report) ClinVar-classified variants reported only for phenotypes/conditions NOT related to cancer, default: False
--debug            Print full docker commands to log, default: False
</pre></div>
</div>
<p>The <em>cpsr</em> software release contains an example VCF file.</p>
<p>Report generation with the example VCF, using the <a class="reference external" href="https://panelapp.genomicsengland.co.uk/panels/245/">Adult solid tumours
cancer
susceptibility</a>
virtual gene panel, can be performed through the following command:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">python</span> <span class="o">~/</span><span class="n">cpsr</span><span class="o">-</span><span class="mf">0.6</span><span class="o">.</span><span class="mi">2</span><span class="o">/</span><span class="n">cpsr</span><span class="o">.</span><span class="n">py</span>
 <span class="o">--</span><span class="n">query_vcf</span> <span class="o">~/</span><span class="n">cpsr</span><span class="o">-</span><span class="mf">0.6</span><span class="o">.</span><span class="mi">2</span><span class="o">/</span><span class="n">example</span><span class="o">.</span><span class="n">vcf</span><span class="o">.</span><span class="n">gz</span>
 <span class="o">--</span><span class="n">pcgr_dir</span> <span class="o">~/</span><span class="n">pcgr</span><span class="o">-</span><span class="mf">0.9</span><span class="o">.</span><span class="mi">2</span>
 <span class="o">--</span><span class="n">output_dir</span> <span class="o">~/</span><span class="n">cpsr</span><span class="o">-</span><span class="mf">0.6</span><span class="o">.</span><span class="mi">2</span>
 <span class="o">--</span><span class="n">genome_assembly</span> <span class="n">grch37</span>
 <span class="o">--</span><span class="n">panel_id</span> <span class="mi">1</span>
 <span class="o">--</span><span class="n">sample_id</span> <span class="n">example</span>
 <span class="o">--</span><span class="n">secondary_findings</span>
 <span class="o">--</span><span class="n">gwas_findings</span>
 <span class="o">--</span><span class="n">classify_all</span>
 <span class="o">--</span><span class="n">maf_upper_threshold</span> <span class="mf">0.2</span>
 <span class="o">--</span><span class="n">no_vcf_validate</span>
</pre></div>
</div>
<p>Note that the example command also refers to the PCGR directory
(<em>pcgr-0.9.2</em>), which here contains the data bundle that are necessary
for both <em>PCGR</em> and <em>CPSR</em>.</p>
<p>This command will run the Docker-based <em>cpsr</em> workflow and produce the
following output files in the <em>output</em> folder:</p>
<ol class="arabic simple">
<li><p><strong>example.cpsr.grch37.vcf.gz (.tbi)</strong> - Bgzipped VCF file with
relevant annotations appended by CPSR</p></li>
<li><p><strong>example.cpsr.grch37.pass.vcf.gz (.tbi)</strong> - Bgzipped VCF file with
relevant annotations appended by CPSR (PASS variants only)</p></li>
<li><p><strong>example.cpsr_config.rds</strong> - CPSR configuration object (RDS format),
mostly for debugging purposes</p></li>
<li><p><strong>example.cpsr.grch37.pass.tsv.gz</strong> - Compressed TSV file (generated
with <a class="reference external" href="https://github.com/sigven/vcf2tsv">vcf2tsv</a>) of VCF content
with relevant annotations appended by CPSR</p></li>
<li><p><strong>example.cpsr.grch37.html</strong> - Interactive HTML report with
clinically relevant variants in cancer predisposition genes organized
into tiers</p></li>
<li><p><strong>example.cpsr.grch37.json.gz</strong> - Compressed JSON dump of HTML report
content</p></li>
<li><p><strong>example.cpsr.grch37.snvs_indels.tiers.tsv</strong> - TSV file with key
annotations of tier-structured SNVs/InDels</p></li>
</ol>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="annotation_resources.html" class="btn btn-neutral float-right" title="CPSR annotation resources" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="about.html" class="btn btn-neutral float-left" title="About" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2018-2021, Sigve Nakken

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
    

  <style>
    /* Sidebar header (and topbar for mobile) */
    .wy-side-nav-search, .wy-nav-top {
      background: #0B6121;
    }
    /* Sidebar */
    /*.wy-nav-side {*/
    /*  background: #ff0000;*/
    /*}*/
  </style>


</body>
</html>