

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Input &mdash; Cancer Predisposition Sequencing Reporter 0.5.1 documentation</title>
  

  
  

  

  
  
    

  

  
  
    <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  

  

  
        <link rel="author" title="About these documents"
              href="about.html"/>
        <link rel="index" title="Index"
              href="genindex.html"/>
        <link rel="search" title="Search" href="search.html"/>
    <link rel="top" title="Cancer Predisposition Sequencing Reporter 0.5.1 documentation" href="index.html"/>
        <link rel="next" title="Output" href="output.html"/>
        <link rel="prev" title="CPSR annotation resources" href="annotation_resources.html"/> 

  
  <script src="_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="index.html" class="icon icon-home"> Cancer Predisposition Sequencing Reporter
          

          
          </a>

          
            
            
              <div class="version">
                0.5.1
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
                <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="about.html">About</a></li>
<li class="toctree-l1"><a class="reference internal" href="getting_started.html">Getting started</a></li>
<li class="toctree-l1"><a class="reference internal" href="annotation_resources.html">CPSR annotation resources</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Input</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#vcf">VCF</a></li>
<li class="toctree-l2"><a class="reference internal" href="#cpsr-configuration-file">CPSR configuration file</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="output.html">Output</a></li>
<li class="toctree-l1"><a class="reference internal" href="variant_classification.html">Variant classification (ACMG)</a></li>
<li class="toctree-l1"><a class="reference internal" href="CHANGELOG.html">CHANGELOG</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="index.html">Cancer Predisposition Sequencing Reporter</a>
      </nav>


      
      <div class="wy-nav-content">
        <div class="rst-content">
          

 



<div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="index.html">Docs</a> &raquo;</li>
      
    <li>Input</li>
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="_sources/input.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="input">
<h1>Input<a class="headerlink" href="#input" title="Permalink to this headline">¶</a></h1>
<p>The CPSR workflow accepts a single input file:</p>
<ul class="simple">
<li>An unannotated, single-sample <strong>VCF file</strong> (&gt;= v4.2) with germline
calls (SNVs/InDels)</li>
</ul>
<div class="section" id="vcf">
<h2>VCF<a class="headerlink" href="#vcf" title="Permalink to this headline">¶</a></h2>
<ul class="simple">
<li>We <strong>strongly</strong> recommend that the input VCF is compressed and
indexed using <a class="reference external" href="http://www.htslib.org/doc/tabix.html">bgzip</a> and
<a class="reference external" href="http://www.htslib.org/doc/tabix.html">tabix</a></li>
<li>If the input VCF contains multi-allelic sites, these will be subject
to <a class="reference external" href="http://genome.sph.umich.edu/wiki/Vt#Decompose">decomposition</a></li>
<li>Variants used for reporting should be designated as ‘PASS’ in the VCF
FILTER column</li>
</ul>
<p><strong>IMPORTANT NOTE</strong>: CPSR generates a number of VCF INFO annotation tags
that is appended to the query VCF. We will therefore encourage the users
to submit query VCF files that have not been subject to annotations by
other means, but rather a VCF file that comes directly from variant
calling. If not, there are likely to be INFO tags in the query VCF file
that coincide with those produced by CPSR.</p>
</div>
<div class="section" id="cpsr-configuration-file">
<h2>CPSR configuration file<a class="headerlink" href="#cpsr-configuration-file" title="Permalink to this headline">¶</a></h2>
<p>The cancer predisposition sequencing report can be flexibly configured
in a TOML-formatted configuration file. The default TOML configuration
file, are shown below:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="c1"># CPSR configuration options (TOML).</span>

<span class="p">[</span><span class="n">secondary_findings</span><span class="p">]</span>
<span class="c1">## Include variants found in ACMG-recommended list for incidental findings</span>
<span class="c1">## https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/</span>
<span class="n">show_sf</span> <span class="o">=</span> <span class="n">true</span>

<span class="p">[</span><span class="n">maf_limits</span><span class="p">]</span>
<span class="c1">## choose upper MAF threshold in gnomAD&#39;s superpopulation (global) for report of unclassified (non-ClinVar)</span>
<span class="c1">## variants</span>
<span class="n">maf_gnomad</span> <span class="o">=</span> <span class="mf">0.05</span>

<span class="p">[</span><span class="n">popgen</span><span class="p">]</span>
<span class="c1">## choose population source in gnomAD (non-cancer subset for use as control), defaults to the global set</span>

<span class="c1">## For gnomaAD, this can by any of the following values (three-letter codes):</span>
<span class="c1">## &quot;afr&quot; - African/American (12,020 individuals (7,652 WES / 4,368 WGS))</span>
<span class="c1">## &quot;amr&quot; - Admixed American (17,210 individuals (16,791 WES / 419 WGS))</span>
<span class="c1">## &quot;eas&quot; - East Asian (9,435 individuals (8,624 WES / 811 WGS))</span>
<span class="c1">## &quot;sas&quot; - Sout Asian (15,391 individuals (15,391 WES / 0 WGS))</span>
<span class="c1">## &quot;nfe&quot; - Non-Finnish European (63,369 individuals (55,860 WES / 7,509 WGS))</span>
<span class="c1">## &quot;fin&quot; - Finnish (12,897 individuals (11,150 WES / 1,747 WGS))</span>
<span class="c1">## &quot;oth&quot; - Other (3,234 individuals (2,743 WES / 491 WGS))</span>
<span class="c1">## &quot;global&quot; - All populations (138,632 individuals (123,136 WES / 15,496 WGS))</span>
<span class="n">pop_gnomad</span> <span class="o">=</span> <span class="s2">&quot;global&quot;</span>

<span class="p">[</span><span class="n">visual</span><span class="p">]</span>
<span class="c1"># Choose visual theme of report, any of: &quot;default&quot;, &quot;cerulean&quot;, &quot;journal&quot;, &quot;flatly&quot;, &quot;readable&quot;, &quot;spacelab&quot;, &quot;united&quot;, &quot;cosmo&quot;, &quot;lumen&quot;, &quot;paper&quot;, &quot;sandstone&quot;, &quot;simplex&quot;, or &quot;yeti&quot; (https://bootswatch.com/)</span>
<span class="n">report_theme</span> <span class="o">=</span> <span class="s2">&quot;default&quot;</span>

<span class="p">[</span><span class="n">custom_tags</span><span class="p">]</span>
<span class="c1">## list VCF info tags that should be present in JSON output</span>
<span class="c1">## tags should be comma separated, i.e. custom_tags = &quot;GATK_FILTER,VARSCAN_FILTER&quot;</span>
<span class="n">custom_tags</span> <span class="o">=</span> <span class="s2">&quot;&quot;</span>

<span class="p">[</span><span class="n">classification</span><span class="p">]</span>
<span class="c1">## for variants with an existing classification (ClinVar), keep CPSR score and classification in output TSV file</span>
<span class="n">clinvar_cpsr</span> <span class="o">=</span> <span class="n">false</span>

<span class="p">[</span><span class="n">gwas</span><span class="p">]</span>
<span class="n">gwas_hits</span> <span class="o">=</span> <span class="n">false</span>
<span class="c1">## Required p-value for reporting of GWAS hits</span>
<span class="n">p_value_min</span> <span class="o">=</span> <span class="mf">5e-8</span>

<span class="p">[</span><span class="n">other</span><span class="p">]</span>
<span class="n">n_vcfanno_proc</span> <span class="o">=</span> <span class="mi">4</span>
<span class="n">n_vep_forks</span> <span class="o">=</span> <span class="mi">4</span>
<span class="n">vep_skip_intergenic</span> <span class="o">=</span> <span class="n">false</span>
<span class="c1">## choice of how VEP chooses the primary transcript pr. gene</span>
<span class="c1">## https://www.ensembl.org/info/docs/tools/vep/script/vep_other.html#pick_options</span>
<span class="n">vep_pick_order</span> <span class="o">=</span> <span class="s2">&quot;canonical,appris,tsl,biotype,ccds,rank,length,mane&quot;</span>
</pre></div>
</div>
</div>
</div>


           </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="output.html" class="btn btn-neutral float-right" title="Output" accesskey="n">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="annotation_resources.html" class="btn btn-neutral" title="CPSR annotation resources" accesskey="p"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2018-2019, Sigve Nakken.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'./',
            VERSION:'0.5.1',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true
        };
    </script>
      <script type="text/javascript" src="_static/jquery.js"></script>
      <script type="text/javascript" src="_static/underscore.js"></script>
      <script type="text/javascript" src="_static/doctools.js"></script>

  

  
  
    <script type="text/javascript" src="_static/js/theme.js"></script>
  

  
  
  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.StickyNav.enable();
      });
  </script>
    

  <style>
    /* Sidebar header (and topbar for mobile) */
    .wy-side-nav-search, .wy-nav-top {
      background: #0B6121;
    }
    /* Sidebar */
    /*.wy-nav-side {*/
    /*  background: #ff0000;*/
    /*}*/
  </style>


</body>
</html>