

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Variant classification (ACMG) &mdash; Cancer Predisposition Sequencing Reporter 2.1.0 documentation</title>
  

  
  
  
  

  

  
  
    

  

  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    <link rel="author" title="About these documents" href="about.html" />
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="CHANGELOG" href="CHANGELOG.html" />
    <link rel="prev" title="Output" href="output.html" /> 

  
  <script src="_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="index.html" class="icon icon-home"> Cancer Predisposition Sequencing Reporter
          

          
          </a>

          
            
            
              <div class="version">
                2.1.0
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="about.html">About</a></li>
<li class="toctree-l1"><a class="reference internal" href="getting_started.html">Getting started</a></li>
<li class="toctree-l1"><a class="reference internal" href="annotation_resources.html">CPSR annotation resources</a></li>
<li class="toctree-l1"><a class="reference internal" href="input.html">Input</a></li>
<li class="toctree-l1"><a class="reference internal" href="output.html">Output</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Variant classification (ACMG)</a></li>
<li class="toctree-l1"><a class="reference internal" href="CHANGELOG.html">CHANGELOG</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Cancer Predisposition Sequencing Reporter</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html">Docs</a> &raquo;</li>
        
      <li>Variant classification (ACMG)</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="_sources/variant_classification.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="variant-classification-acmg">
<h1>Variant classification (ACMG)<a class="headerlink" href="#variant-classification-acmg" title="Permalink to this headline">¶</a></h1>
<p>The unclassified, coding variants in the CPSR workflow have all been
assigned a composite pathogenicity score (coined <strong>PATHSCORE</strong> in the
tables below) according to the prioritization scheme outlined in
<a class="reference external" href="https://github.com/ding-lab/charger/">CharGer</a> (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/29625052">Huang et al., Cell,
2018</a>), as well as
<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/27153395">Maxwell et al., Am J Hum Genet,
2016</a>. Specifially, a
cancer-specific adoption of previously established ACMG criteria for
variant classification has been implemented (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/25741868">Richards et al.,
2015</a>, <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/27181684">Amendola et
al., Am J Hum Genet,
2016</a>). The following
ACMG evidence items indicate support for pathogenic/benign variants, and
have been implemented as follows (points in parenthesis indicate how
they contribute to overall <strong>PATHSCORE</strong>):</p>
<ul class="simple">
<li><em>PVS1</em> (<strong>8</strong>) - null variant (nonsense, frameshift, canonical ±1 or
2 splice sites - indicated by <a class="reference external" href="https://github.com/konradjk/loftee">VEP’s LofTee
plugin</a>) in a gene where LoF
is a known mechanism of disease (<em>dominant</em> mode of inheritance
(MoI))</li>
<li><em>PS1</em> (<strong>7</strong>) - Same amino acid change as a previously established
pathogenic variant
(<a class="reference external" href="https://www.ncbi.nlm.nih.gov/clinvar/">ClinVar</a>) regardless of
nucleotide change</li>
<li><em>PSC1</em> (<strong>4</strong>) - null variant (nonsense, frameshift, canonical ±1 or
2 splice sites - indicated by <a class="reference external" href="https://github.com/konradjk/loftee">VEP’s LofTee
plugin</a>) in a gene where <em>LoF</em>
is a known mechanism of disease (<em>recessive</em> MoI)</li>
<li><em>PMC1</em> (<strong>2</strong>) - null variant (nonsense, frameshift, canonical ±1 or
2 splice sites - indicated by <a class="reference external" href="https://github.com/konradjk/loftee">VEP’s LofTee
plugin</a>) in a gene where <em>LoF</em>
is NOT the known mechanism of disease</li>
<li><em>PM1</em> (<strong>2</strong>) - Located in a somatic mutational hotspot
(<a class="reference external" href="https://www.cancerhotspots.org">cancerhotspots.org</a>)</li>
<li><em>PM2</em> (<strong>2</strong>) - Absence/extremely low minor allele frequency (MAF &lt;
0.0005 in <a class="reference external" href="http://www.internationalgenome.org/">1000 Genomes
Project</a>/<a class="reference external" href="http://gnomad.broadinstitute.org/">gnomAD</a>
global population)</li>
<li><em>PM4</em> (<strong>2</strong>) - Protein length changes due to inframe indels or
stoploss variants in non-repetitive regions (as identified by
<a class="reference external" href="http://www.repeatmasker.org/">RepeatMasker</a>) of known
susceptibility gene (<em>dominant</em> MoI)</li>
<li><em>PM5</em> (<strong>2</strong>) - Novel missense change at an amino acid residue where
a different missense change determined to be pathogenic has been seen
before (<a class="reference external" href="https://www.ncbi.nlm.nih.gov/clinvar/">ClinVar</a>)</li>
<li><em>PP2</em> (<strong>1</strong>) - Missense variant in a gene that has a relatively low
rate of benign missense variation and where missense variants are a
common mechanism of disease. Current settings:<ul>
<li>Rate of benign missense variants in susceptibility gene is &lt; 20%</li>
<li>Rate of non-truncating pathogenic missense variants in
susceptibility gene is &gt; 50%</li>
</ul>
</li>
<li><em>PP3</em> (<strong>1</strong>) - Multiple lines of computational evidence support a
deleterious effect on the gene or gene product (conservation,
evolutionary, splicing impact, etc., from
<a class="reference external" href="https://sites.google.com/site/jpopgen/dbNSFP">dbNSFP</a>)</li>
<li><em>PPC1</em> (<strong>1</strong>) - Protein length changes due to inframe indels or
stoploss variants in non-repetitive regions (as identified by
<a class="reference external" href="http://www.repeatmasker.org/">RepeatMasker</a>) of known
susceptibility gene (<em>recessive</em> MoI)</li>
<li><em>BP4</em> (<strong>-1</strong>) - Multiple lines of computational evidence support a
benign effect on the gene or gene product (conservation,
evolutionary, splicing impact, etc., from
<a class="reference external" href="https://sites.google.com/site/jpopgen/dbNSFP">dbNSFP</a>)</li>
<li><em>BMC1</em> (<strong>-2</strong>) - Peptide change is at the same location (codon) of a
known benign change
(<a class="reference external" href="https://www.ncbi.nlm.nih.gov/clinvar/">ClinVar</a>)</li>
<li><em>BSC1</em> (<strong>-6</strong>) - Peptide change is known to be benign
(<a class="reference external" href="https://www.ncbi.nlm.nih.gov/clinvar/">ClinVar</a>)</li>
<li><em>BA1</em> (<strong>-8</strong>) - High allele frequency in the general population (MAF
&gt; 0.05 in <a class="reference external" href="http://www.internationalgenome.org/">1000 Genomes
Project</a>/<a class="reference external" href="http://gnomad.broadinstitute.org/">gnomAD</a>
global population)<ul>
<li>Exception for <a class="reference external" href="https://www.ncbi.nlm.nih.gov/gene/3077">homeostatic iron regulator
(HFE)</a> and
<a class="reference external" href="https://www.ncbi.nlm.nih.gov/gene/5265">SERPINA1</a>, requiring
MAF &gt; 0.25</li>
</ul>
</li>
</ul>
<p>For <em>PP3</em> and <em>BP4</em> CPSR requires by default that a minimum of five
distinct algorithms must constitute a majority vote (damaging/benign),
and that maximally one algorithm votes for the minority
(damaging/benign)</p>
<p>The composite <strong>PATHSCORE</strong> is finally assigned one of four different
<strong>PATHRANK</strong> levels:</p>
<ul class="simple">
<li>HIGH: <em>PATHSCORE</em> &gt; <strong>8</strong></li>
<li>MODERATE: <em>PATHSCORE</em> &lt;= <strong>8</strong> AND <em>PATHSCORE</em> &gt; <strong>4</strong></li>
<li>LOW: <em>PATHSCORE</em> &lt;= <strong>4</strong> AND <em>PATHSCORE</em> &gt;= <strong>0</strong></li>
<li>BENIGN: <em>PATHSCORE</em> &lt; <strong>0</strong></li>
</ul>
<p>The contribution of ACMG evidence items pr. variant can be seen in the
<strong>PATHDOC</strong> variable.</p>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="CHANGELOG.html" class="btn btn-neutral float-right" title="CHANGELOG" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="output.html" class="btn btn-neutral" title="Output" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2018, Sigve Nakken.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'./',
            VERSION:'2.1.0',
            LANGUAGE:'None',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true,
            SOURCELINK_SUFFIX: '.txt'
        };
    </script>
      <script type="text/javascript" src="_static/jquery.js"></script>
      <script type="text/javascript" src="_static/underscore.js"></script>
      <script type="text/javascript" src="_static/doctools.js"></script>

  

  <script type="text/javascript" src="_static/js/theme.js"></script>

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>  

  <style>
    /* Sidebar header (and topbar for mobile) */
    .wy-side-nav-search, .wy-nav-top {
      background: #0B6121;
    }
    /* Sidebar */
    /*.wy-nav-side {*/
    /*  background: #ff0000;*/
    /*}*/
  </style>


</body>
</html>