## CPSR annotation resources

### Basic variant consequence annotation
  * [VEP v98.3](http://www.ensembl.org/info/docs/tools/vep/index.html) - Variant Effect Predictor ([GENCODE v31](https://www.gencodegenes.org/human/) as gene reference database (v19 for grch37))

###  *Insilico* predictions of effect of coding variants
  * [dBNSFP](https://sites.google.com/site/jpopgen/dbNSFP) - database of non-synonymous functional predictions (v4.0, May 2019)

###  Variant frequency databases
  * [gnomAD](http://exac.broadinstitute.org/) - germline variant frequencies exome-wide (r2.1, October 2018)
  * [dbSNP](http://www.ncbi.nlm.nih.gov/SNP/) - database of short genetic variants (build 152/152, January 2019)
  * [Cancer Hotspots](http://cancerhotspots.org) - a resource for statistically significant mutations in cancer (2017)

### Variant databases of clinical utility
  * [ClinVar](http://www.ncbi.nlm.nih.gov/clinvar/) - database of clinically related variants (November 2019)
  * [CIViC](https://civicdb.org) - clinical interpretations of variants in cancer (November 5th 2019)

### Protein domains/functional features
  * [UniProt/SwissProt KnowledgeBase](http://www.uniprot.org) - resource on protein sequence and functional information (2019_10, November 2019)
  * [Pfam](http://pfam.xfam.org) - database of protein families and domains (v32, September 2018)

### Cancer gene knowledge bases
  * [CancerMine](https://zenodo.org/record/3525385#.XcHblUVKiL4) - Literature-mined database of tumor suppressor genes/proto-oncogenes (v18, November 2019)
  * [Genomics England PanelApp](https://panelapp.genomicsengland.co.uk) - cancer phenotype panels as of November 16th 2019


### Phenotype ontologies
  * [MedGen](https://www.ncbi.nlm.nih.gov/medgen/)
